lib/pterodactyl_service.php

```diff
@@ -38,7 +38,7 @@ public function addUserParameters(array $vars)
         Loader::loadModels($this, ['Clients']);
         $client = $this->Clients->get($vars['client_id']);
         return [
-            'username' => $client->email,
+            'username' => $client->first_name . '_' . $client->last_name . '_' . $client->id,
```

Comment:
I think the username should be something that is unlikely to change. The client's name could change in Blesta, but not here. The name may also contain spaces or apostrophes, etc., so if th ename were used, it should be stripped for only valid characters.

Maybe something like `"blesta-" . $client->id`?